src/auth.rs

```diff
@@ -17,6 +22,66 @@ pub fn parse_auth(auth: String, host: &str) -> io::Result<(String, Option<String
     }
 }
 
+fn get_home_dir() -> Option<PathBuf> {
+    #[cfg(all(target_os = "win", test))]
```

Comment:
The binary used for the integration tests unfortunately isn't compiled in test mode, that's also why the other `XH_TEST_MODE` variables are compiled unconditionally.